plugins/stack-overflow-backend/src/search/StackOverflowQuestionsCollatorFactory.ts

```diff
@@ -80,7 +83,11 @@ export class StackOverflowQuestionsCollatorFactory
     const baseUrl =
       config.getOptionalString('stackoverflow.baseUrl') ||
       'https://api.stackexchange.com/2.2';
-    return new StackOverflowQuestionsCollatorFactory({ ...options, baseUrl });
+    return new StackOverflowQuestionsCollatorFactory({
+      ...options,
+      baseUrl,
+      config,
```

Comment:
@emmaindal and I have a suggestion, how about getting the `apiKey` from the configuration here and passing it as `apiKey`? Similar to what we did with `baseUrl`?